types/types.go

```diff
@@ -256,7 +256,10 @@ type Info struct {
 	Runtimes           map[string]Runtime
 	DefaultRuntime     string
 	Swarm              swarm.Info
-	LiveRestore        bool
+	// Determine whether containers should be kept running when the
```

Comment:
Please follow the [Go style guidelines](https://golang.org/doc/effective_go.html#commentary) when documenting fields.